client/src/components/Pagination/Pagination.js

```diff
@@ -21,8 +21,10 @@ export default class Pagination extends React.Component {
 
     componentDidUpdate(prevProps, prevState, snapshot) {
         const { count, page } = this.props;
-        if (count !== prevProps.count) this.setState({count});
-        if (page !== prevProps.page) {
+        if (count !== prevProps.count && !isNaN(count)) {
```

Comment:
Were you running into NaNs?

Reply:
Yes, something to do with the values defaulting to strings I think...